apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -521,6 +521,36 @@
                 }
             },
             {% endif %}
+            {% if console.applications_settings is defined %}
+            "applicationsSettings": {
```

Comment:
Lets not define a new feature config for this. Rather add a `disabledFeature` in the `existing `applications` config.

```
"applications" :{
    "disabledFeatures": [
         "application.settings.dcr"
     ] 
}
```

Reply:
Changed with [69dcc96](https://github.com/wso2/identity-apps/pull/5964/commits/69dcc96fdd841b5618b3b3cac223428e1f45305b)